moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -532,9 +538,21 @@ KinematicState Servo::getNextJointState(const ServoInput& command)
     {
       RCLCPP_DEBUG_STREAM(logger_, "Joint velocity limit scaling applied by a factor of " << joint_limit_scale);
     }
-
     target_state.velocities *= joint_limit_scale;
```

Comment:
Naming, to be consistent...

```suggestion
    target_state.velocities *= joint_velocity_limit_scale;
```